interop/tapir/src/test/scala/caliban/interop/tapir/TestData.scala

```diff
@@ -22,13 +23,25 @@ object TestData {
     case class Mechanic(shipName: String) extends Role
   }
 
-  case class Character(name: String, nicknames: List[String], origin: Origin, role: Option[Role])
+  case class Character(
+    name: String,
+    nicknames: List[String],
+    origin: Origin,
+    role: Option[Role],
+    labels: UIO[List[String]] = ZIO.succeed(Nil)
+  )
 
   case class CharactersArgs(origin: Option[Origin])
   case class CharacterArgs(name: String)
 
   val sampleCharacters = List(
-    Character("James Holden", List("Jim", "Hoss"), EARTH, Some(Captain("Rocinante"))),
+    Character(
+      "James Holden",
+      List("Jim", "Hoss"),
+      EARTH,
+      Some(Captain("Rocinante")),
+      labels = ZIO.succeed(List("Captain"))
```

Comment:
Is it possible to delay this effect and test that the initial payload arrives immediately when this is deferred or would that be too difficult?

Reply:
That might be slightly tricky so I'd say we could do that as a follow up. My thought is that we would need to record when we receive each event and verify that at least a certain amount of time has passed between events